### PR TITLE
The Great Purge™

### PR DESCRIPTION
**Pull request by p3lim**
A lot of things going on in this commit, as discussed internally within the team.
This is in preparation of the next major release (7.0.0).

- Changed all files to follow a set styleguide (which will be published)
- Renamed all (relevant) elements to follow a certain naming scheme
- Removed deprecations
- Removed CPoints element (functionality is within ClassIcons, and this eliminates #240)
- Misc cleanup

In response to #321

### CODE REVIEW

File: elements/aura.lua [hunk 1]
```diff
@@ -129,20 +127,20 @@ local createAuraIcon = function(icons, index)
...
-local updateIcon = function(unit, icons, index, offset, filter, isDebuff, visible)
+local function updateIcon(unit, icons, index, offset, filter, isDebuff, visible)
 	local name, rank, texture, count, dispelType, duration, expiration, caster, isStealable,
 		nameplateShowSelf, spellID, canApply, isBossDebuff, casterIsPlayer, nameplateShowAll,
 		timeMod, effect1, effect2, effect3 = UnitAura(unit, index, filter)
 
 	if(name) then
-		local n = visible + offset + 1
-		local icon = icons[n]
+		local i = visible + offset + 1
```
Comment (Rainrider) [inline elements/aura.lua]: Missed renaming n to i at line 252. github won't let me comment there 😞

Comment (p3lim) [inline elements/aura.lua]: `index` is not suitable, `offset` makes sense but it's already used in this context.
`iconIndex` or `adjustedOffset` would be the best.

Comment (Rainrider) [inline elements/aura.lua]: or position?

Comment (Rainrider) [inline elements/aura.lua]: I "fixed" it silently to i on the docs branch, so that I can test later without having to rebase.

Comment (p3lim) [inline elements/aura.lua]: Yeah sorry, I had fixed it just forgot to push. I need to rebase the entire PR anyways.

**Comment by p3lim** [comment]
Rebased to master and fixed one variable in `elements\auras.lua`.

**Pull request merged**